static/admin/config.yml

```diff
@@ -55,24 +55,6 @@ collections:
         valueField: "key"
         searchFields: ["name"]
         multiple: true
-  - name: "theoryCategories"
```

Comment:
If we remove this the "category" selection on the article won't work anymore.

If you would like to not show the categories in the CMS, you can add `hide: true` here

Reply:
@yannisgu thank you for the clarification. This is indeed what I wanted to do: to hide the categories in the CMS, not the remove them entirely.